ginivision/src/androidTest/java/net/gini/android/vision/test/JpegByteArraySubject.java

```diff
@@ -0,0 +1,120 @@
+package net.gini.android.vision.test;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.common.truth.FailureStrategy;
+import com.google.common.truth.Subject;
+import com.google.common.truth.SubjectFactory;
+
+import net.gini.android.vision.test.exif.ExifReader;
+import net.gini.android.vision.test.exif.ExifReaderException;
+
+class JpegByteArraySubject extends Subject<JpegByteArraySubject, byte[]> {
```

Comment:
I'm not sure I understand the "Subject" reference. I wouldn't know what to expect of this class without reading the code.